data/json/professions.json

```diff
@@ -1989,6 +2324,45 @@
       "female": { "entries": [ { "item": "bra" }, { "item": "panties" } ] }
     }
   },
+  {
```

Comment:
This one has potential, though I haven't cross-checked against the full list of professions.